src/app/common/validation/forms/amount-validators.ts

```diff
@@ -47,6 +47,7 @@ export function btcInsufficientBalanceValidator({
     .typeError(FormErrorMessages.MustBeNumber)
```

Comment:
> Perhaps calling the validation again on press of the Continue button?

but the validation already runs again on continue, no?

Haven't tested but if the empty field/must be defined errors only show when pressing _Continue_ I think that's fine?

Reply:
The validation isn't re-running on submit so it lets you pass an undefined amount:

https://github.com/hirosystems/wallet/assets/2938440/c3b74c53-ae05-4b51-ab5e-5d69c205ae13



I'll see if that's easy to fix, it probably is

Reply:
what if `skipAbsent: formik.submitCount > 0` 🤔

Reply:
I don't understand how the validation wouldn't be running on submit when I see the line in the code when routing to choose the fee? 🤔 

`await formikHelpers.validateForm();`

Reply:
I see that @fbwoolf but when that code runs it doesn't seem to block anything. It continues to the next step with `undefined` values, and then it can crash. 

I tried using `formik.submitCount === 0` and it seemed to work initially but not if you return `Back`. I'll keep investigating anyway.